src/components/HearingOutcome/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { AmendmentArrValues, AmendmentValues, IndividualAmendmentArrValues } from "types/Amendments"
+
+const isAmendmentArrValue = (value: AmendmentValues, indexToUpdate: number): value is AmendmentArrValues =>
+  (value as AmendmentArrValues)[indexToUpdate].updatedValue !== undefined
+
+export const isAmendmentArr = (value: AmendmentValues): value is AmendmentArrValues => typeof value !== "string"
```

Comment:
```suggestion
export const isAmendmentArr = (value: AmendmentValues): value is AmendmentArrValues => Array.isArray(value)
```

Would this be more explicit?